src/components/CalendarBodyForMonthView.tsx

```diff
@@ -124,11 +124,15 @@ function _CalendarBodyForMonthView<T>({
                       return [
                         ...elements,
                         index > maxVisibleEventCount - 1 ? (
-                          <Text style={{ fontSize: 11, marginTop: 2, fontWeight: 'bold' }}>
+                          <Text
+                            key={index}
```

Comment:
Using an index as a key can cause problems if the order of elements changes. Maybe it will be ok in this case, but if there some member of the `event` instance that can be used to uniquely identify that would be a better option.

Reply:
@glenn-axsy I've made a contribution to this PR which addresses your concern above.
Hopefully it's ok for me to approve in your absence today